cs/configuring.texy

```diff
@@ -107,6 +108,28 @@ $configurator->addParameters([
 ]);
 \--
 
+Lze definovat i dynamické parametry, které nezpůsobí přegenerování containeru.
+
+/--php
+$configurator->addDynamicParameters([
+	'rootDir' => 'foobar',
+]);
+\--
+
+
+Přidání služeb
+--------------
+
+Služby můžeme do DI containeru přidat i ručně.
```

Comment:
Co nějak takto:

Vložení vlastní instance
--------------

 Do DI containeru můžeme vložit vlastní instanci třídy, namísto toho, aby jí vytvářel sál. Službu definujeme s příznakem `dynamic: yes`. 

 /--neon
 services:
 	myservice:
 		class: App\Model\MyCustomService
 		dynamic: yes
 \--

A poté v bootrapu vložíme do kontaineru:

 /--php
 $configurator->addServices([
 	'myservice' => new App\Model\MyCustomService('foobar'),
 ]);
 \--